eventfiles/views.py

```diff
@@ -42,3 +44,20 @@ def download_files(request):
             response['Content-Disposition'] = 'inline;filename=' + path.basename(file_path)
             return response
     raise Http404
+
+
+@login_required(login_url='welcome_page')
```

Comment:
can you change it to 'home'?
'welcome page' is the first screen before login,
but 'home' is the screen after the login has happened

Reply:
Done 😄